diagnostic.rb

```diff
@@ -2,13 +2,42 @@
 # Diagnostic answers to:
 
 # question 2:
+Licensee.create(
+  surname: 'Bishop',
+  given_name: 'Caren',
+  gender: 'f',
+  height: 63,
+  weight: 132,
+  born_on: '1943-09-26',
+  eye_color: 'Brown',
+  hair_color: 'Black'
+)
 
 # question 3:
 Licensee.transaction do
   CSV.foreach 'data/licensees.csv', headers: true do |row|
+    Licensee.create(
+      surname: row[0],
+      given_name: row[1],
+      gender: row[2],
+      height: row[3],
+      weight: row[4],
+      born_on: row[5],
+      eye_color: row[6],
+      hair_color: row[7]
+    )
   end
 end
 
 # question 4:
```

Comment:
There are a number of methods to use SQL, example being .find_by_sql or .count_by_sql, ect. But if you're using SQL here is probably a better way using active record.  I see what you're trying to do, but active record is easier and better for this case.